aip/general/0133.md

```diff
@@ -60,8 +60,8 @@ rpc CreateBook(CreateBookRequest) returns (Book) {
 - There **should** be exactly one `google.api.method_signature` annotation,
   with a value of `"parent,{resource},{resource}_id"`, or "`"parent,{resource}"`
   if the resource ID is not required.
-- If the API is operating on the [management plane][], the operation should have
-  [strong consistency][]: the completion of a create operation **must** mean
+- If the API is operating on the [management plane](https://google.aip.dev/111#management-plane), the operation should have
```

Comment:
```suggestion
- If the API is operating on the [management plane][], the operation should have
```
Revert this, please. The change you've made isn't necessary, this already properly linked:
![Screenshot 2023-09-12 at 9 13 52 AM](https://github.com/aip-dev/google.aip.dev/assets/6644735/caf6cb0f-8c7c-4d41-b471-c44c6d5f4ee5)